main.py

```diff
@@ -14,8 +14,8 @@
 
 from pydantic.v1 import BaseModel
 
-from croptalk.model_llm import model
-# from croptalk.model_openai_functions import model
+# from croptalk.model_llm import model
+from croptalk.model_openai_functions import model
```

Comment:
That change should already be in `openai_functions` branch by now, so you just need to pull changes from it